packages/core/src/browser/style/tooltip.css

```diff
@@ -0,0 +1,28 @@
+/********************************************************************************
+ * Copyright (C) 2021 Arm and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+.theia-tooltip {
+    color: var(--theia-list-hoverForeground) !important;
+    background: var(--theia-list-hoverBackground) !important;
+    border: 1px solid !important;
+    border-color: var(--theia-list-hoverForeground) !important;
```

Comment:
We already make use of the shorthand `border`, we can simplify to:

```suggestion
    border: 1px solid var(--theia-list-hoverForeground) !important;
```

Reply:
That approach didn't work! For some reason the variable didn't expand properly and splitting the css rules was the only way I got it to work :/

Reply:
Just tried again and it won't work :/

<img width="632" alt="Screenshot 2021-09-15 at 18 18 27" src="https://user-images.githubusercontent.com/61341/133479768-9be50db1-d961-4041-a660-6c8dc9bac17e.png">